src/root/common.tt

```diff
@@ -414,7 +414,18 @@ BLOCK renderEvals %]
         [% END %]
         <th style="width: 10em">Date</th>
         <th>Input changes</th>
-        <th colspan='2' style="width: 25em">Success</th>
+        <th style="width: 2em">
+        Jobs<br>
+            <img src="[% c.uri_for("/static/images/emojione-check-2714.svg") %]" height="12" width="12" title="Succeeded" alt="Succeeded" class="build-status" />
+        </th>
+        <th style="width: 2em">
+            <img src="[% c.uri_for("/static/images/emojione-red-x-274c.svg") %]" height="12" width="12" title="Failed" alt="Failed" class="build-status" />
+        </th>
+        <th style="width: 2em">
+            <img src="[% c.uri_for("/static/images/emojione-question-2754.svg") %]" height="12" width="12" title="Queued" alt="Queued" class="build-status" />
+
```

Comment:
Be mindful not to add empty lines where there were none.

(Same for the other instance in the file.)

Reply:
Fixed.